### PR TITLE
Anna Terelius project 11

### PR DESCRIPTION
**Pull request by AnnaTerelius**
Link to my deployed project:
https://annatereliusproject11happythoughts.netlify.com/

My reflections on how this week's project turned out:
I'm happy I managed to get this done, I really struggle to understand what's happening in the code

Things I'd like to have clarified or explained in more detail:
I can't really Point at one specific thing, more or less the hole thing..hard to grasp..

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,72 @@
...
+
+  useEffect(() => {
+    fetch('https://technigo-thoughts.herokuapp.com/')
+    .then (res => res.json())
+    .then (json => setThoughts(json));
+  }, []);
+
+  const handleFormSubmit = (event) => {
+    event.preventDefault()
+    console.log('event handleFormsubmit= när man submittar ny text')
```
Comment (ericaglimsholt) [inline code/src/App.js]:
```suggestion

```

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,72 @@
...
+      body: '',
+      headers: { 'Content-Type': 'application/json' }
+    })
+    .then((res) => res.json())
+    .then((json) => console.log(json))
+   .then(() => onLiked(_id))
+  }
+  // nedan skapar en lokal "storage" för antal likes innan man uppdaterar sidan.
+  const onLiked = (thoughtId) => {
+    console.log('testing onLiked')
```
Comment (ericaglimsholt) [inline code/src/App.js]:
```suggestion

```

File: code/src/App.js [hunk 3]
```diff
@@ -1,9 +1,72 @@
...
+    fetch(`https://technigo-thoughts.herokuapp.com/${_id}/like`, {
+      method: 'POST',
+      body: '',
+      headers: { 'Content-Type': 'application/json' }
+    })
+    .then((res) => res.json())
+    .then((json) => console.log(json))
+   .then(() => onLiked(_id))
+  }
+  // nedan skapar en lokal "storage" för antal likes innan man uppdaterar sidan.
```
Comment (ericaglimsholt) [inline code/src/App.js]: Write comments in English

**Comment by ericaglimsholt** [comment]
Note: A good idea is to test out if the design works when there's a lot of text

image

**Comment by AnnaTerelius** [comment]
Hi!
Thank you, yes I didn’t know how to make it fit when there is a lot of text, how should I do?

I was going to do some changes for exampel the swedish text in my code but I got merge conflicts from Git hub when trying to push the new stuff so I didn’t know what to do, any tips?

I tried to talk to my team but they also thought it was a hard week to understand..
/Anna

Från: Erica Glimsholt <notifications@github.com>
Skickat: den 11 december 2019 14:25
Till: Technigo/project-happy-thoughts <project-happy-thoughts@noreply.github.com>
Kopia: AnnaTerelius <tereliusanna@hotmail.com>; Author <author@noreply.github.com>
Ämne: Re: [Technigo/project-happy-thoughts] Anna Terelius project 11 (#25)


Note: A good idea is to test out if the design works when there's a lot of text

[Bild som tagits bort av avsändaren. image]<https://eur04.safelinks.protection.outlook.com/?url=https%3A%2F%2Fuser-images.githubusercontent.com%2F22143776%2F70625109-02395900-1c22-11ea-9a56-0189b46a0811.png&data=02%7C01%7C%7C2d64088a10434950c0f208d77e3d91c3%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C637116675204318164&sdata=NtrhScycHf%2B7b3gi68eB2j20kV7%2F6sGAo2zgsnljENk%3D&reserved=0>

**Pull request closed without merging**